build_projects/dotnet-host-build/PrepareTargets.cs

```diff
@@ -244,6 +273,11 @@ public static BuildTargetResult RestorePackages(BuildTargetContext c)
                 .Execute()
                 .EnsureSuccessful();
 
+            dotnet.Restore("--verbosity", "verbose", "--disable-parallel")
+                .WorkingDirectory(Path.Combine(c.BuildContext.BuildDirectory, "tools", "dotnet-deb-tool"))
```

Comment:
How about just restore from the "tools" directory, so we don't need 2 calls here?

Also, `RepoRoot` instead of `BuildDirectory`.  I think that reads cleaner.